package/MDAnalysis/coordinates/DCD.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, filename, convert_units=True, dt=None, **kwargs):
         self.skip_timestep = self._file.header['nsavc']
 
         self._ts_kwargs['dt'] = dt
-        self.ts = self._Timestep(self.n_atoms, **self._ts_kwargs)
+        self.ts = base.Timestep(self.n_atoms, **self._ts_kwargs)
```

Comment:
I didn't delete that already?

Reply:
Shouldn't we also delete `self._Timestep` then?

Reply:
yeah self._Timestep is a thing of the past